### DISCUSSION TITLE
ExplorerPatcher crashes Windows Explorer

### DISCUSSION THREAD

**Discussion by manniacmind** [valinet/ExplorerPatcher#2911] [General]
So, I recently had the same problem as others, that Windows Defender claimed EP was some sort of virus and blocked the app.
I unblocked it and installed the new 63.4 version, because this was suggested in the discussions.

But today Windows Explorer started to.. not work. I'm not able to start it while EP is running.
I also tried installing the older 63.2 version but Windows Explorer still does not work. I tried resetting EP settings to the DEFAULT SETTINGS, but still no change.
After uninstalling EP, Windows Explorer works again.

Anyone else having this problem or a suggestion how to fix it?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

Please include what settings you changed in explorer patcher properties application that you can remember.

Please include any other software you have installed that modifies the UI of windows.

**Reply by pyrates999** [reply]
It's also a false flag.  Have you updated windows defender signatures?

**Reply by manniacmind** [reply]
Windows 11 23H2 (22631.3155)

i did not change any setting. these are EP default settings.

i have also installed openshell

**Reply by manniacmind** [reply]
sure, updated the signatures an hour ago. but, as i said, i have already unblocked EP from windows defender.

**Comment by Amrsatrio** [comment]
Have you ever used ViVeTool to disable Tabs or any other feature?

**Reply by manniacmind** [reply]
i don't know what that is

**Reply by Amrsatrio** [reply]
If you uninstall OpenShell, does Explorer work again? I don't test EP with OpenShell regularly, sorry about that.

**Reply by manniacmind** [reply]
when I uninstall EP, explorer works. why would i uninstall an unrelated program?

**Reply by Amrsatrio** [reply]
I wanted to know if this issue is isolated to ExplorerPatcher or OpenShell. Also, I don't know if it helps, but try downloading ViVeTool and doing `vivetool /query > result.txt` and sending the result.txt here.

**Reply by manniacmind** [reply]
Update: The issues seems to have gone after a couple of reboots.
Thanks for the help!
